doc/sphinx-guides/source/user/find-use-data.rst

```diff
@@ -62,7 +62,11 @@ Once on a dataset page, you will see the title, citation, description, and sever
 View Files
 ----------
 
-Files in Dataverse each have their own page that can be reached through the search results or through the Files table on their parent dataset's page. The dataset page and file page offer much the same functionality in terms of viewing and editing files, with a few small exceptions. The file page includes the file's persistent identifier (DOI or handle), which can be found under the Metadata tab. Also, the file page's Versions tab gives you a version history that is more focused on the individual file rather than the dataset as a whole. 
+Files in Dataverse each have their own page that can be reached through the search results or through the Files table on their parent dataset's page. The dataset page and file page offer much the same functionality in terms of viewing and editing files, with a few small exceptions. 
```

Comment:
Sometimes we say "landing page". We could say "their own landing page" if we want. No big deal.

Reply:
added in https://github.com/IQSS/dataverse/commit/a992e0efe0ab817bc0afec5b9bf355868440e72a

Reply:
Talked about in standup, this is all set.